imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePickerConfig.java

```diff
@@ -8,11 +8,13 @@
 import com.esafirm.imagepicker.features.imageloader.ImageLoader;
 import com.esafirm.imagepicker.model.Image;
 
+import java.io.*;
```

Comment:
I think this could be `java.io.File` only

Reply:
Yeah, apologies. Bad habit to use the IDE's auto import when working on other people's projects.